src/can-deploy.ts

```diff
@@ -124,11 +110,14 @@ export class CanDeploy {
 
 export default (options: CanDeployOptions) => new CanDeploy(options);
 
-export interface CanDeployOptions extends SpawnArguments {
-  participant?: string;
-  participantVersion?: string;
-  to?: string;
-  latest?: boolean | string;
```

Comment:
...and this?

Reply:
I've just realised you marked changes requested - did you want me to make it backwards-compatible?

Reply:
No, I think your other point answered this. We'll just leave them behind and upgrade Pact Node to the next major revision.